drivers/metal/README.md

```diff
@@ -0,0 +1,39 @@
+# Metal Rendering Device
+
+This document aims to describe the Metal rendering device implementation in Godot.
+
+## Future work / ideas
+
+* Use placement heaps
+* Explicit hazard tracking
+* [MetalFX][] upscaling support?
```

Comment:
Missing URL.

Reply:
Ah I see it's at the bottom of the file. Does that syntax worked in GitHub Flavored Markdown?

Reply:
Fixed